ChangeLog.md

```diff
@@ -8,14 +8,27 @@ Shoop Change Log
   releasing next version, the "Unreleased" header will be replaced with
   appropriate version header and this help text will be removed.
 
-- Create the Shop as active with `shoop_init` management command
-- Fix usages of `Category.get_ancestors()` in templates
-- Remove Stripe integration (shoop.stripe)
+- Add a basic REST API for reading/writing products and reading orders (merge 7a31a88)
+- Use the database to store shopping baskets by default (merge 458d6ef)
+- Implement pluggable shopping basket storage backends (merge 458d6ef)
+- Implement basic contact group admin (merge 185bf1c)
+- Add telemetry (usage statistics) system (merge c42e243)
+- Add Dockerfile (merge 737666d)
+- Improve admin login flow (merge 8bb9e05)
+  ReadTheDocs (merge 92e7959)
+- Document settings; make documentation builds available on
```

Comment:
Damnit. :D You see, I had originally written this in forward chronological order, then realized it should be reverse, and just used "reverse lines"...